projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/confirm-flag-status/confirm-flag-status.component.spec.ts

```diff
@@ -1,12 +1,14 @@
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { FormGroup, ReactiveFormsModule } from '@angular/forms';
-
+import { CaseFlagFieldState, ConfirmStatusErrorMessage } from '../../enums';
 import { ConfirmFlagStatusComponent } from './confirm-flag-status.component';
 
 describe('ConfirmFlagStatusComponent', () => {
   let component: ConfirmFlagStatusComponent;
   let fixture: ComponentFixture<ConfirmFlagStatusComponent>;
+  let nextButton: any;
```

Comment:
can this be of type Element | null ?

Reply:
Good spot, @MateiHMCTS - I'll change it to `HTMLElement` and do the same in other test classes